terraform/stacks/regionalmasterbosh/stack.tf

```diff
@@ -56,6 +56,7 @@ resource "aws_lb" "main" {
     bucket  = var.log_bucket_name
     prefix  = var.stack_description
     enabled = true
+  enable_deletion_protection  = true
```

Comment:
This line is in the wrong place. It needs to outside of this `access_logs` block, above current line 55